crowd_anki/representation/json_serializable.py

```diff
@@ -6,9 +6,12 @@
 
 class JsonSerializable:
     readable_names = {}
-    export_filter_set = {"mod",  # Modification time
-                         "usn",  # Todo clarify
-                         "id"}
+    export_filter_set = {
+        "mod",  # Modification time
+        "usn",  # Todo clarify
+        "id",
+        "import_config"
```

Comment:
why do we need this?

Reply:
Because the import config (the final one that is created at the end of the user import selection window) is added to the deck, and then passed down to the notes and whatnot. This is so that each can check it for their own desired import option, such as notes which look for the personal_field settings.

Reply:
Ahh, yes you are correct! :grin: I am only using the import_config values inside the individual save_to_collection functions anyways, so this is a much better solution. Changed :+1: